src/get-started/java-binding.md

```diff
@@ -162,17 +162,21 @@ See [Serialization](#serialization) for details.
 
 !!! note "Example of ProofMapIndex Creation"
     ```java
-    void updateBalance(Fork fork) {
-      var name = "balanceById";
+    void updateBalance(Fork fork, String serviceInstanceName) {
+      var name = serviceInstanceName + ".balanceById";
       var balanceById = ProofMapIndexProxy.newInstance(name, fork,
           StandardSerializers.hash(),
           StandardSerializers.longs());
       balanceById.put(id, newBalance);
     }
     ```
 
-A set of named collections constitute a *service scheme*. For convenient access
-to service collections you can implement a factory of service collections.
+A set of named collections constitute a *service schema*. Collections shall be
+defined in a unique to all service instances namespace to avoid name collisions;
+that is usually achieved by adding a prefix to the name of each collection, 
+declared in the service (e.g., an assigned service name or numeric id:
+"timestamping.timestamps"). For convenient access to service collections
```

Comment:
Nit: `"timestamping.timestamps"` should use code formatting. Also, since `Prefixed` accesses is already merged in the code, I'd suggest to mention service name only instead of service name or numeric ID.